Monika After Story/game/script-topics.rpy

```diff
@@ -15985,27 +15985,34 @@ label monika_eating_meat:
     m 3eud "If you're asking if I would do so for {i}survival{/i}, I wouldn't hesitate. {w=0.2}It's not that eating meat is distressing for me or anything."
     m 7eud "I told you before, I'm vegetarian because of the impact of the mass-production of meat on the environment...{w=0.2}{nw}"
     extend 2euc "which also includes fish farming, so I'm not a pescatarian."
-    m 2rsc "...That said, I don't consider myself vegan either. {w=0.3}{nw}"
-    extend 4eud "Sure, the consumption of animal products contributes to environmental damage, but a lot of vegan alternatives have their own issues too..."
-    m 4euc "These include things like importing perishable products across great distances and mass-farming in conditions that are both cruel for workers and a strain on the local ecosystem."
+    m 2rsc "...That said, I don't consider myself vegan either."
+    m 7eua "I understand a lot of vegans are so not just for health or environmental reasons, but also ethical ones, so I think that's commendable in its own way."
+    m 7eksdlb "But I'm more worried about the environmental part of it personally. "
```

Comment:
We typically don't `extend` without a pause. But this case I don't think needs the `extend`.
```suggestion
    m 7eksdlb "But I'm more worried about the environmental part of it personally..."
```